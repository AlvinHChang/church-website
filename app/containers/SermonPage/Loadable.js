/**
 * Asynchronously loads the component for SermonPage
 */

import loadable from 'utils/loadable';

export default loadable(() => import('./index'));
