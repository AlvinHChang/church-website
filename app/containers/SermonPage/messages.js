/*
 * SermonPage Messages
 *
 * This contains all the text for the SermonPage container.
 */
import { defineMessages } from 'react-intl';

export const scope = 'app.containers.SermonPage';

export default defineMessages({
  header: {
    id: `${scope}.header`,
    defaultMessage: 'This is the SermonPage container!',
  },
});
