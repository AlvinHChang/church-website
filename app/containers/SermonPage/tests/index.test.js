import React from 'react';
import { render } from 'react-testing-library';
import { IntlProvider } from 'react-intl';

import SermonPage from '../index';

describe('<SermonPage />', () => {
  it('should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
      <IntlProvider locale="en">
        <SermonPage />
      </IntlProvider>,
    );
    expect(firstChild).toMatchSnapshot();
  });
});
