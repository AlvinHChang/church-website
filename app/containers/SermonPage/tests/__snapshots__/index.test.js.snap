// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SermonPage /> should render and match the snapshot 1`] = `
<h1>
  <span>
    This is the SermonPage container!
  </span>
</h1>
`;
